lib/data/network_services/network_service_impl.dart

```diff
@@ -50,7 +50,7 @@ class NetworkServiceImpl extends NetworkService {
       }
 
       result.add(
-        TrackModel(artistNames: artistNames, songName: songName),
+        TrackModel(artistNames: artistNames, songName: [songName]),
```

Comment:
why songName is a list?

Reply:
I do not remember, why songName is a list, but I fixed that